event_sale_registration_multi_qty/__manifest__.py

```diff
@@ -1,15 +1,13 @@
-# -*- coding: utf-8 -*-
-# Copyright 2017 David Vidal <david.vidal@tecnativa.com>
-# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# Copyright 2017 Tecnativa - David Vidal
```

Comment:
Here and the next line should cover year 2017-2019. @chienandalu apologies if asking again but maybe GitHub has some issues to keep my comments